.github/ISSUE_TEMPLATE/feature-request.yml

```
@@ -0,0 +1,82 @@
+---
+name: ✨  Feature Request
+description: Suggest an idea for this project
+labels: ["feat/enhancement ", "needs-triage"]
+
+body:
+- type: markdown
+  attributes:
+    value: >
+      **Thank you for wanting to suggest a feature for Cookiecutter!**
```

Again, we're not adding the function to the original `cookiecutter` -
```suggestion
      **Thank you for wanting to suggest a feature for cookiecutter-python!**
```